projects/rpx-xui-translation/src/lib/rpx-translation.service.ts

```diff
@@ -153,18 +117,26 @@ export class RpxTranslationService {
 
     this.requestTimerSubscription = timer(this.config.debounceTimeMs).subscribe(() => {
       const url = this.config.baseUrl + `/${lang}`;
-      const s = this.http.post<TranslationsDTO>(url, { phrases: this.requesting[lang] }).pipe(
-        map(t => t.translations),
-        catchError(() => {
-          const translations: { [from: string]: string } = {};
-          this.requesting[lang].forEach(p => translations[p] = this.config.testMode ? `[Test translation for ${p}]` : p);
-          return of(translations);
-        })
-      ).subscribe((translations: { [from: string]: TranslationModel }) => {
+      const s = this.http.post<TranslationsDTO>(url,
+        {
+          phrases: this.requesting[lang],
+        }
+      )
+        .pipe(
+          map(t => t.translations),
+          catchError(() => {
+            const translations: { [from: string]: string } = {};
+            this.requesting[lang].forEach(p => translations[p] = this.config.testMode ? `[Test translation for ${p}]` : p);
+            return of(translations);
+          })
+      ).subscribe((translations: { [from: string]: string }) => {
```

Comment:
translations is now [string]: object

Reply:
API doesn't seem to return this. It currently returns { [from: string]: string }